draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -800,125 +787,54 @@ Note that none of these forms alter the HTTPS origin or authority.
-of this procedure.
-
-Conversion from HTTPSSVC to Alt-Svc Field Value SHOULD ignore any
-unrecognized SvcParamKeys, and conversion from Alt-Svc Field Value to
-HTTPSSVC SHOULD ignore any Alt-Svc parameters that do not have a
-corresponding SvcParamKey.
-
-For example, if the operator of https://www.example.com
-intends to include an HTTP response header like
-
-    Alt-Svc: h3="svc.example.net:8003"; ma=3600; foo=123, \
-             h2="svc.example.net:8002"; ma=3600; foo=123
-
-they could also publish an HTTPSSVC DNS RRSet like
-
-    www.example.com. 3600 IN HTTPSSVC 2 svc.example.net. (
-                                        alpn=h3 port=8003 foo=123 )
-                             HTTPSSVC 3 svc.example.net. (
-                                        alpn=h2 port=8002 foo=123 )
-
-Where "foo" is a hypothetical future HTTPSSVC and Alt-Svc parameter.
-
-This data type can also be represented as an Unknown RR as described in
-{{!RFC3597}}:
-
-    www.example.com. 3600 IN TYPE??? \\# TBD:WRITEME
+    uri-host ":" port
 
-On connections to an HTTPSSVC alternative service, clients SHOULD
-include the same Alt-Used header that they would include if the
-corresponding Alt-Svc Field Value were received over HTTPS.
+where uri-host is the final value of HOST ({client-behavior}) minus
+the trailing ".", and port is the port number in use.
 
 
-## Differences from Alt-Svc as transmitted over HTTP
+## Differences from Alt-Svc
 
-Publishing an alternative services form HTTPSSVC record in DNS is intended
-to be equivalent to transmitting the corresponding Alt-Svc value over
-HTTPS, and receiving an HTTPSSVC record is intended to be equivalent to
-receiving this field value over HTTPS.  However, there are some small
+Publishing a ServiceForm HTTPSSVC record in DNS is intended
+to be similar to transmitting the corresponding Alt-Svc field value over
+HTTPS, and receiving an HTTPSSVC record is intended to be similar to
+receiving that field value over HTTPS.  However, there are some
 differences in the intended client and server behavior.
 
-### Max Age and Persist
-
-There is no SvcParamKey corresponding to the Alt-Svc "ma" (max age) parameter.
-Instead, server operators SHOULD encode the
-expiration time in the DNS TTL, and MUST NOT set a TTL longer than the
-intended "max age".
-
-For security reasons, there is no SvcParamKey corresponding to the Alt-Svc
-"persist" parameter.
-
-## Multiple records and preference ordering {#pri}
-
-Server operators MAY publish multiple ServiceForm HTTPSSVC
-records as an RRSet.  When converting a collection of alt-values
-into an HTTPSSVC RRSet, the server operator MUST set the
-overall TTL to a value no larger than the minimum
-of the "max age" values (following Section 5.2 of {{!RFC2181}}).
+### Insecure delivery
 
-Each RR corresponds to exactly one alt-value, as described
-in Section 3 of {{!AltSvc}}.
+SVCB does not ensure secure delivery of the SvcFieldValue, so any Alt-Svc
```

Comment:
Do we need to define "secure delivery"?  That term seems hand-wavy.

Reply:
True.  (So does "untrusted channels" below.)  I guess we're really talking about authenticity?  But maybe the thing to do is to repeat "untrusted channels" here...

Reply:
Perhaps be consistent about "untrusted channels" but then define what that means somewhere, such as its first reference?

Reply:
OK, I've rephrased this to be about "untrusted channels".  Please review.